server/utils/trim.ts

```diff
@@ -1,5 +1,6 @@
-export default function trimForm<T>(form: Record<string, unknown>): T {
+export default function trimForm<T extends Record<string, unknown>>(form: T): T {
```

Comment:
I attempted to sort out the typescript error but then encountered another one so gave up and ignored it